docs/serving/configuration/config-defaults.md

```diff
@@ -30,319 +32,497 @@ data:
-and the system default is used.
-Below is an example of setting revision-cpu-limit.
-By default, it is not set by Knative.
+The `revision-cpu-limit` value determines the default CPU allocation limit for revisions. If this value is omitted, the system default is used. This key is not enabled by default for Knative.
 {% endraw %}
 
-**Key**: `revision-cpu-limit`
-
-**Default**: `"1000m"` (1 CPU (aka 1000 milli-CPU))
+* **Global key:** `revision-cpu-limit`
+* **Per-revision annotation key:** `serving.knative.dev/revision-cpu-limit`
+* **Possible values:** integer
+* **Default:** `"1000m"` (1 CPU, or 1000 milli-CPU)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  revision-cpu-limit: "1000m" # 1 CPU (aka 1000 milli-CPU)
-```
 
-### Revision Memory Limit
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      revision-cpu-limit: "1000m"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/revision-cpu-limit: "1000m"
+        spec:
+          containers:
+            - image: gcr.io/knative-samples/helloworld-go
+    ```
+
+### Revision memory limit
 {% raw %}
-revision-memory-limit contains the memory allocation to limit
-revisions to by default.  If omitted, no value is specified
-and the system default is used.
-Below is an example of setting revision-memory-limit.
-By default, it is not set by Knative.
+The `revision-memory-limit` value determines the default memory allocation limit for revisions. If this value is omitted, the system default is used. This key is not enabled by default for Knative.
 {% endraw %}
 
-**Key**: `revision-memory-limit`
-
-**Default**: `"200M"` (200 megabytes of memory)
+* **Global key:** `revision-memory-limit`
+* **Per-revision annotation key:** `serving.knative.dev/revision-memory-limit`
+* **Possible values:** integer
+* **Default:** `"200M"` (200 megabytes of memory)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  revision-memory-limit: "200M" # 200 megabytes of memory
-```
+
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      revision-memory-limit: "200M"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/revision-memory-limit: "200M"
```

Comment:
Likewise this annotation doesn't exist - the config map property configures the   default value for `Spec.Containers[*].Limits.Memory` of a `Revision`

Reply:
similar to above